### PR TITLE
New inventory page design.

### PR DESCRIPTION
**Pull request by Moo64c**
See #107.

### CODE REVIEW

File: garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module [hunk 1]
```diff
@@ -256,7 +259,7 @@ function garmentbox_inventory_get_summary($node) {
     $defective = $wrapper->field_defective->value();
     // Determine if we need to add or subtract the quantity from the total
     // calcualtion.
-    $in_warehouse = $wrapper->field_inventory_type->field_in_warehouse->value();
+    $in_warehouse = $wrapper->field_inventory_type->field_in_warehouse->value() && !$defective;
```
Comment (Moo64c) [inline garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module]: Wrong.

File: garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module [hunk 2]
```diff
@@ -264,16 +267,22 @@ function garmentbox_inventory_get_summary($node) {
...
     $include_in_total = $inventory_types[$tid]['include_in_total'] && !$defective;
 
+    $is_closed = $wrapper->field_inventory_type->field_closed_stock->value() && !$defective;
+
     $inventory[$nid]['label'] = $wrapper->field_item_variant->label();
     if (!$defective) {
       $type_key = $tid;
       $inventory[$nid]['types'][$type_key]['type_label'] = $wrapper->field_inventory_type->label();
       $inventory[$nid]['types'][$type_key]['in_total'] = $include_in_total;
+      $inventory[$nid]['types'][$type_key]['in_warehouse'] = $in_warehouse;
```
Comment (Moo64c) [inline garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module]: Add `&& !$defective`

File: garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module [hunk 3]
```diff
@@ -283,6 +292,7 @@ function garmentbox_inventory_get_summary($node) {
       $inventory[$nid]['total'][$term->tid]['quantity'] = !empty($inventory[$nid]['total'][$term->tid]['quantity']) ? $inventory[$nid]['total'][$term->tid]['quantity'] : 0;
     }
 
+
```
Comment (Moo64c) [inline garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module]: Not needed.

File: garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module [hunk 4]
```diff
@@ -329,9 +340,9 @@ function garmentbox_inventory_get_summary($node) {
     }
   }
 
+  $size_amount_width = 2;
```
Comment (Moo64c) [inline garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module]: Documentation.

File: garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module [hunk 5]
```diff
@@ -393,12 +407,43 @@ function garmentbox_inventory_get_summary($node) {
...
 /**
+ * Get the correct class to add to a row of variant status display.
+ *
+ * @param @variables
+ *   Array containing order status term variables.
+ *
+ * @return
+ *   String; Class to add.
+ */
+function _garmentbox_inventory_quantity_box_color($variables) {
```
Comment (amitaibu) [inline garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module]: `_garmentbox_inventory_subcell_class()`

File: garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module [hunk 6]
```diff
@@ -413,12 +458,20 @@ function garmentbox_inventory_theme($existing, $type, $theme, $path) {
...
         ),
       ),
-      'template' => 'inventory-summary-cell',
+      'template' => 'templates/inventory-summary-cell',
+    ),
+    'garmentbox_inventory_sub_cell' => array(
+      'variables' => array(
+        'ordered' => NULL,
+        'quantity' => NULL,
+        'quantity_color' => NULL,
```
Comment (amitaibu) [inline garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module]: class

File: garmentbox/modules/custom/garmentbox_inventory/templates/inventory-summary-cell.tpl.php [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<div class="inventory-cell clearfix">
+  <div class="main-figure">
+    <span class="amount"><?php print $available; ?> </span>
+    <div class="label">available</div>
```
Comment (amitaibu) [inline garmentbox/modules/custom/garmentbox_inventory/templates/inventory-summary-cell.tpl.php]: `print t('available');`

File: garmentbox/modules/custom/garmentbox_inventory/templates/inventory-summary-sub-cell.tpl.php [hunk 8]
```diff
@@ -0,0 +1,13 @@
+<div class="inventory-sub-cell clearfix">
+  <?php if ($quantity): ?>
+    <div class="main-figure <?php print $quantity_color; ?>">
+      <span class="amount"><?php print $quantity; ?></span>
+    </div>
+
+    <?php if ($ordered): ?>
+      <div class="sub-figure">
+        <div class="ordered"><?php print $ordered; ?> <span class="label">ordered</span></div>
```
Comment (amitaibu) [inline garmentbox/modules/custom/garmentbox_inventory/templates/inventory-summary-sub-cell.tpl.php]: t()

File: garmentbox/themes/garmentbox_omega/sass/_mixins.scss [hunk 9]
```diff
@@ -113,3 +113,77 @@
     visibility: hidden;
   }
 }
+
+$default_rounded_amount: 5px;
```
Comment (amitaibu) [inline garmentbox/themes/garmentbox_omega/sass/_mixins.scss]: Check [Compass](http://compass-style.org/reference/compass/css3/border_radius/)

File: garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module [hunk 10]
```diff
@@ -393,12 +412,43 @@ function garmentbox_inventory_get_summary($node) {
...
+  $warehouse = $variables['in_warehouse'];
+  $closed = $variables['closed'];
+
+  if ($total) {
+
+    if(!$warehouse && !$closed) {
+      return 'future';
+    }
+
+    if ($closed xor $warehouse) {
```
Comment (amitaibu) [inline garmentbox/modules/custom/garmentbox_inventory/garmentbox_inventory.module]: Add docs.

File: garmentbox/themes/garmentbox_omega/sass/_mixins.scss [hunk 11]
```diff
@@ -1,3 +1,11 @@
+/**
```
Comment (amitaibu) [inline garmentbox/themes/garmentbox_omega/sass/_mixins.scss]: Remove

**Pull request merged**